Finished settings and dashboard pages

haldarritam opened the pull request:
@michellegasto ,

Felipe and I have coded this branch together, and we both agree that this is ready to be 🚢 from a technical perspective. 

Could you please pull this branch and run it on your phone and let us know if it meets your requirements and design intentions? Also, could you please navigate through the screens and see if you find any bug or crash and let us know?

Thank you.

michellegasto commented:
This branch looks good. I think my only comment was that 1) the second button on the Settings-Features page was still labeled "Set weekly..." as opposed to "Vibration Alert".

haldarritam commented:
@michellegasto Thanks a lot for reviewing the changes. We have modified the code and added some new features to it. When you have time, please review the changes and let us know what you think.

michellegasto commented:
Everything looks good to me. I had no issues entering info or moving through the different settings options. The hard coded info on the dashboard page works well. The progress bar for the sensor looks good and having the error message when trying to start a session without a sensor connected was useful. Great job!
